modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,246 @@
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+
+
+def calc_coords(bb):
```

Comment:
I think we can remove `calc_coords`, because `xmax`,`ymax` are used only to calculate `IoU` and `cx, cy` are used to calculate `Precision`. We can calculate necessary coords in corresponding function.

Reply:
I disagree, because in with `calc_coords` we calculating corners and centers of the bounding boxes just once. If we split calculations between other functions it is gonna be more complex. Example: for now we calculate xmax, xmin, ymax, ymin once for every frame. If we will remove `calc_coords` we would be forced to calculate it three times in every method.

Reply:
If we removed this function, then we calculated `xmax, ymax` in IoU and `cx, cy` in `get_precision(bbox, gt, is_norm=False)`. Thus, calculation will not be more complicated.

Reply:
By "complex" I mean the large number of calculations, sorry for inaccuracy in the previous message.
So, if we will remove `calc_coords` we would be forced to calculate `xmax, xmin, ymax, ymin` three times (in every method) and `cx, cy`  two times (for precision and normalized precision) for _every frame_. And we have 280 videos with at least 1000 frames each. I do not think it is rational.

Reply:
Now, we call `calc_coords `in a loop:
```python
# For every frame in video
for number_of_the_frame, image in enumerate(video_sequence):
```
calculated for each frame, right?
> three times (in every method) 
What are these methods? Why are `ymax, ymin`  needed in `get_precision(bbox, gt)`?

Reply:
Yes, `calc_coords` called twice for each frame: for ground truth and for prediction.
You are right: `xmax, xmin, ymax, ymin` are not needed for precision/normalized precision - my mistake.
I will think about how `get_iou`, `get_pr` and `get_norm_pr` methods can be reworked.
But, I still think combining `get_pr` and `get_norm_pr` can affect flexibility of the benchmark.